src/core/agent/services/signifyNotificationService.ts

```diff
@@ -422,11 +422,11 @@ class SignifyNotificationService extends AgentService {
               break;
             }
             case OperationPendingRecordType.Oobi: {
-              if (!pendingOperation.metadata?.connectionId) {
+              if (!operation.response || !(operation.response as any).i) {
                 break;
               }
               const connectionRecord = await this.connectionStorage.findById(
-                pendingOperation.metadata?.connectionId
+                (operation.response as any).i
```

Comment:
We can just do `operation.response?.i` I think, and remove the if statement and break above to simplify

Reply:
I think we still need this `(operation.response as any)` because `operation.response` type is unknown and I got this `Property 'i' does not exist on type '{}'` error if I just do `operation.response?.i`